dali/operators/image/crop/bbox_crop.cc

```diff
@@ -323,7 +332,13 @@ The values are:
 .. note::
   If left empty, depending on the number of dimensions ``"WH"`` or ``"WHD"`` will be assumed.
 )code",
-        TensorLayout{""});
+        TensorLayout{""})
+    .AddOptionalArg<bool>(
+        "output_bbox_indices",
+        R"code(If set to True, an extra output ``bbox_indices`` will be returned, containing
```

Comment:
```suggestion
        R"code(If set to True, an extra output will be returned, containing
```
I don't we have named outputs.